### DISCUSSION TITLE
Breaking change for EqualException(object, object) missing in release docs for 2.5.0

### DISCUSSION THREAD

**Discussion by LosManos** [xunit/xunit#2761] [General]
In version 2.4.1 it is possible to call
`throw new EqualException(expected, actual);`
as it leads to `EqualException.cs/EqualException`
While in 2.5.0 the class `EqualException` is marked as `internal`.

I believe rewriting the call to 
`throw EqualException.ForMismatchedValues(expected, actual);`
is the correct solution.

I would consider this a breaking change and it should be noted in the release notes.

**Comment by bradwilson** [comment]
Emphasis mine, from https://xunit.net/releases/v2/2.5.0

> Aside from normal quality of life improvements, the primary feature of this release is a heavily updated assertion library, which includes several new assertions, several new overloads for existing assertions, and a pass to ensure that the messages from the assertions are now more concise and consistent. We wrote up an overview of [the updates in the 2.5.0 assertion library messages](https://xunit.net/releases/v2/2.5.0-assertions). **For the most part, developers should not notice much compile-time differences (aside from perhaps the occasional type ambiguity that may come from new overloads); there are also some breaking binary changes, most prominently with the gutting and reworking of the exception classes that are thrown by the individual assertions. Because of some of these breaking changes, we felt it was warranted to bump the version up to 2.5.0.**

You could argue that the wording should be strengthened, but you cannot argue that it's not mentioned.

**Comment by LosManos** [comment]
Correct. Sorry for taking your time. Thx for the help.
